string dotFramePath
string rgbFramePath
string depthFramePath
string poseFilePath
---
string objectName
